example/rest-api/art-institute/art-institute.php

```diff
@@ -84,6 +84,8 @@ function register_aic_block(): void {
 				],
 			],
 		],
+		'required_query' => 'search_art',
```

Comment:
I've gone back and forth with making this a list of queries, and calling it `required_queries` instead, so it's future proofed. But, the question I have is that what could the use case be? One use case that comes to mind:

- The list of items is derived from a search query
- The image for each item can be derived from another query, as the size can be customized

But is it worth the effort to do that now? We would likely need to tweak our consumption of those queries to account for that, and there would be more changes as a result of it? 

So for now, I just left it like this to keep it simple. Happy to get feedback on this and change it.

Reply:
> But is it worth the effort to do that now?

IMO it's not worth it. We don't have defined use cases or feedback requesting this. Even if we did, I would first suggest that users just duplicate the "parent" query and make the necessary adjustments to target the "derived" data.

If the resulting code duplication truly becomes unacceptable, we can work to add an abstraction. But as written here, this abstraction is not intuitive and may cause more confusion.

FWIW, I ran into the same issues with https://github.com/Automattic/remote-data-blocks/pull/371, where the devx of composing two queries together is extremely unintuitive.